rust/benchmarks/src/bin/tpch.rs

```diff
@@ -386,7 +386,7 @@ fn create_logical_plan(ctx: &mut ExecutionContext, query: usize) -> Result<Logic
         //     where
         //         l_shipdate >= date '1994-01-01'
         //         and l_shipdate < date '1994-01-01' + interval '1' year
-        //         and l_discount between 0.06 - 0.01 and 0.06 + 0.01
+        //         and l_discount between .06 - 0.01 and .06 + 0.01
```

Comment:
I don't think it really matters, but my reading of the TPCH spec (validation query) has `0.06` rather than `.06`. 

http://tpc.org/tpc_documents_current_versions/pdf/tpc-h_v2.18.0.pdf

![Screen Shot 2021-02-09 at 5 47 20 PM](https://user-images.githubusercontent.com/490673/107438673-dd2ec780-6afe-11eb-97e2-c14d805fba5e.png)

Reply:
Hm weird. I generated the queries some time ago in here: https://github.com/ballista-compute/sqlparser-rs/tree/main/tests/queries/tpch . maybe the code to generate the queries has been changed slightly

Reply:
Just ran https://github.com/databricks/tpch-dbgen again, it still generates the values as `.06`

Reply:
I think at the time I went through and copied them from the TPC-H document directly to ensure they were 100% aligned.